android/src/main/java/com/myapplication/MainActivity.kt

```diff
@@ -2,15 +2,16 @@ package com.myapplication
 
 import App
 import android.os.Bundle
+import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
-import androidx.appcompat.app.AppCompatActivity
+import cube
 
-class MainActivity : AppCompatActivity() {
+class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
         setContent {
-            App()
+            App(mesh = cube)
```

Comment:
To run and display the sample mesh I just randomly picked the one that is already defined in the project.